### ISSUE TITLE
Bad formatting in text/plain part of notification email.

### ISSUE THREAD

**Issue by nibrahim** [pythonindia/junction#468]
I read email using an offline MUA and prefer to read the text/plain part (rather than the HTML one). It's very badly formatted right now. All the tags used for emphasis etc. are all converted into literals and it's a mess. It looks like the default is text/html and the it's just been quickly converted into text/plain. Here's what it looks like now.

```
"&lt;p&gt;@Reviewer 13&lt;/p&gt;

&lt;p&gt;A few more Flake8 plugin ideas:&lt;/p&gt;

&lt;ol&gt;
&lt;li&gt;Check if function names are &quot;nice&quot; -- cat.py
should only have functions starting with
&lt;code&gt;meow_&lt;/code&gt;&lt;/li&gt;
&lt;li&gt;Weird boss wants all functions in cat.py to accept at least
3 args -- check if cat.py adheres to weird boss&#39;s wish.&lt;/li&gt;
&lt;li&gt;Weird boss wants all functions in cat.py to call
&lt;code&gt;meow_logger()&lt;/code&gt; before proceeding -- check if
cat.py adheres to weird boss&#39;s wish.&lt;/li&gt;
&lt;/ol&gt;

&lt;p&gt;:-) &lt;/p&gt;
"
-------------------------------------------------------------------------------
PS: Please, don't reply to the email. You can
login(http://in.pycon.org/cfp/accounts/login/?next=/cfp/2016/proposals/hacking-the-python-ast~b6W7d/)to
and leave a comment
```

This needs to be cleaned up.

**Comment by kracekumar** [comment]
Yes, we send only `HTML` email. Will add `text` as well.

**Comment by nibrahim** [comment]
Thanks. Maybe I'm just very behind the times preferring text/plain emails. :)

**Comment by kracekumar** [comment]
There are a group of people use it with Emacs, just that in recent changes, I overlooked it.

**Comment by ChillarAnand** [comment]
@nibrahim Sorry for inconvenience.  Will be fixed in #470
